kawilliams8/Museum
# Remove boilerplate html

#### What does this PR do?

Cleans up some boilerplate html in the HelloWorld component, but more importantly, is checking that git and Vue are setup correctly. The Vue CLI and UI installations have been problematic on both of my machines.

#### How should this be manually tested?

Need to clone on the iMac to check git tracking is working correctly. Run test suite.

#### What are the relevant tickets?

N/A

#### Screenshots (if appropriate)

N/A

#### Questions:

None at this time.
